src/service/NemgenService.ts

```diff
@@ -50,24 +50,38 @@ export class NemgenService {
             '--resources=/server-config',
             '--nemesisProperties=./server-config/block-properties-file.properties',
         ];
+        if (presetData.nemgenUseTemporaryCacheDatabase) {
+            cmd.push(`--useTemporaryCacheDatabase`);
+        }
```

Comment:
why did you add option for this? when would you want to run WITHOUT it?

Reply:
Only for testing? By providing: `nemgenUseTemporaryCacheDatabase: true` on custom preset.

Reply:
I have never "read" the cache database, if you think nobody will, I can remove the option and always use temp

Reply:
i think it is always acceptable to use this option. but you can check with @gimre-xymcity.

Reply:
@gimre-xymcity would somebody ever use the cache database for debugging?

Reply:
nope

Reply:
Removed the option